frontend/src/components/navbar/Navbar.jsx

```diff
@@ -47,7 +46,7 @@ export default function Navbar() {
         <LinkList
           name={"Data"}
           categories={{
-            Rondes: { icon: faBicycle, link: "#" },
+            Planningen: { icon: faBicycle, link: "/beheer/planningen/1" },
```

Comment:
This breaks when this schedule is ever removed

Reply:
This is intended. If we want a decent fix for this we need to do it in a larger PR. At the moment there are multiple cases such as this one.